### PR TITLE
create my example hue

### PR DESCRIPTION
**Pull request by erindcole**
## High level description of this Pull-request
Made a folder called hues and an example hue

## Related Issues/Enhancements/Bugs/etc.
#18 

## List all External APIs that are added / removed / renamed
- NA

## List all important changes made 
-added a folder to store hues
-made a file with a hue

## Pending tasks (optional)
- If there are things that must have been implemented as part of this PR but couldn't be done due to dependencies or other reasons, mention them here. This need not be an exhaustive list.
- However, it is expected that developers add TODO comments in the code where needed indicating pending tasks.

## Reviewers
@KirstieJane , @sandan , @brennacole 

# CHECKLIST:
Make sure all items are marked when you submit the pull-request.

- [ x] Relevant documentation checked in and updated.
- [ x] Necessary unit/integration tests added
- [x ] All unit tests passed with no errors
- [x ] All integration tests passed with no errors

### CODE REVIEW

File: hues/0001.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Sensitive and Alone
```
Comment (KirstieJane) [inline hues/0001.md]: I think you could format this as a title by adding the `# ` at the beginning of the line.

> # Sensitive and Alone

If you want to make this heading a little smaller than go for `### ` 😸

**Comment by KirstieJane** [comment]
Want me to add my hue in #18 to this pull request? Or should I just add it after this PR is merged?

File: hues/0001.md (hunk 1 above)
Comment (erindcole) [reply hues/0001.md]: sounds good. Thanks for the addition.

**Comment by erindcole** [comment]
@KirstieJane how about you add your Hue after I merge? Thanks for the insights, and I will make the changes you suggest.

**Pull request merged**